### PR TITLE
Update SDK with latest changes

### PR DESCRIPTION
**Pull request by monica-m-ps**
**What's New?**

**API**
- `SetupIntentsApi`

**Model**
- `CaptureStatus`
- `CreateSetupIntentRequest`
- `EventType`
- `MerchantInitiatedTransactionType`
- `PaymentStatusResponse`
-  `TransactionSource`

**Enhancements**
- Updated the API spec with the latest changes
- Edited the readme.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# dojo-php
+# Dojo PHP SDK
 
-The official [Dojo][dojo] PHP library, supporting Remote Payments API.
+The official [Dojo][dojo] PHP library, supporting [Remote Payments API][api-docs].
```
Comment (aaronforinton-dojo) [inline README.md:3]:
```suggestion
The official [Dojo][dojo] PHP library, supporting the [Payments API][api-docs].
```

Comment (aaronforinton-dojo) [reply README.md:3]: Not sure what we're calling this API going forward, but it's Payments on all the docs for now

**Review by aaronforinton-dojo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (monica-m-ps) [reply README.md]: Right. Fixed. Thanks!

**Review by RomanKondratyev — APPROVED** [review]
No new tests added?

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Running Tests
+
+To run these tests, you will need to install PHPUnit. You can do this by running `composer install` from the root of the project.
+
+Once PHPUnit is installed, you can run the tests by running the following command from the root of the project:
+
+```bash
+    ./vendor/bin/phpunit ./tests/<ApiFactoryTest.php>
```
Comment (RomanKondratyev) [inline tests/README.md:8]: Could be just `./vendor/bin/phpunit ./tests`

**Review by RomanKondratyev — APPROVED** [review]

**Pull request merged**